packages/components/visually-hidden/tests/visually-hidden.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from "@yamada-ui/test"
+import { VisuallyHidden } from "../src"
+
+describe("<VisuallyHidden />", () => {
+  test("VisuallyHidden renders correctly", async () => {
+    render(<VisuallyHidden>Test</VisuallyHidden>)
+    const el = screen.getByText("Test")
+    expect(el.textContent).toBe("Test")
```

Comment:
Wouldn't `toBeInTheDocument()` or `toEqual()` be better?

```ts
expect(el.textContent).toEqual("Test")
// or
expect(el.textContent).toBeInTheDocument()
```

Reply:
This PR is to raise the version and does not require a review. If there is an improvement, please create another issue and discuss it.